CONTRIBUTING.md

```diff
@@ -129,6 +129,32 @@ Use the following conventions when creating a page:
+
+| Term              | Terms to Avoid | Notes |
+|-------------------|----------------|-------|
+| API, APIs         | API's, RPC | Application Programming Interface, a set of functions and definitions for software to connect to other software. |
+| core server, core XRP Ledger server | `rippled` | The `rippled` name is probably going to be retired in the near future, so it's better to refer to it by the more generic name. When necessary, refer to `rippled` in all lowercase and code font. (It's pronounced "ripple dee", and the "d" stands for "daemon" per UNIX tradition.)
+| financial institution | bank, FI, PSP (payment services provider) | This term encompasses a wider range of businesses than just "bank" or other terms and does not rely on an understanding of industry jargon. |
+| ledger entry      | ledger object, node | A single object inside the state data of the XRP Ledger. The term "ledger object" could refer to one of these or to the whole ledger. The term "node" was sometimes used for this case because the ledger's state data can be envisioned as a graph, but this is confusing because "node" has other uses. |
+| liquidity provider | market maker | A business or individual who offers to exchange between two currencies or assets, often deriving income from the price differential between trades. The term "market maker" has a specific legal definition in some jurisdictions and may not apply in all the same circumstances. |
+| malicious actor   | hacker | A person, organization, or even an automated tool which might attempt to acquire secrets, break encryption, deny service, or otherwise attack a secure resource. |
+| a NFToken         | NFT, an NFToken | A NFToken object in the XRP Ledger tracks or represents a non-fungible token. Pronounced "nif-token" and written "a NFToken" rather than "an NFToken" |
+| PostgreSQL        | Postgres | A specific brand of relational database software. Always use the full name, not an informal short version. |
+| order book        | orderbook, offer book | A collection of trade orders waiting to be matched & executed, typically sorted by exchange rate. Use two words. |
+| server            | node | A server is software and/or hardware, especially the ones that connect to the XRP Ledger peer-to-peer network. The term "node" is sometimes used for this purpose but is also overloaded with other meanings including entries in a graph and Node.js, a JavaScript interpreter. |
+| stablecoin issuer | gateway | An issuer is the organization that issues a token in the XRP Ledger. A stablecoin is a token where the issuer promises that it is fully backed by some outside asset (such as fiat currency), with the stablecoin issuer providing deposit and withdraw operations to convert between the two (possibly for a fee). Previously the term "gateway" was used (especially by Ripple, the company) to describe this use case, but the rest of the industry adopted "stablecoin issuer" instead. |
```

Comment:
An _issuer_ is the organizaton... A _stablecoin_ is a token... Previously, the term _gateway_ was used... adopted _stablecoin issuer_